fulp_modules/features/species/beefmen/mob/beefman.dm

```diff
@@ -210,22 +210,22 @@
 
 /datum/species/beefman/get_species_lore()
 	return list(
-		"On a very quiet day, the Russian-famous Fiddler Diner was serving food to the crew, when they realized they ran out of burger ingredients. \
-		After drawing straws, the Cook was sent to fetch some more meat from the Morgue, unaware of the events that will transpire. \
-		'It's normal for the Kitchen to grab dead bodies, right? It's not like they need them... Right?' The Cook thought, \
-		inattentively grabbing the first body they could find, trying to get this over with before it becomes a memory. \
-		What the Cook hadn't noticed, the Morgue's tray was green, the body was filled with a soul, one that was begging not to be gibbed.",
-
-		"The Cook one'd and two'd the body into the gibber and turned it on, the grinder struggling to keep up on its unupgraded parts. \
-		Once the whole body entered the machine, it suddenly stopped working, and instead started spitting the meat back out, as if it was in reverse... \
-		The Cook looked over to see what was going on, but the slab of meat looked back, with massive eyes.",
-
-		"After a quick confiscation from the Russian Sol Government, most records of 'Beefmen' have gone dark, \
-		with minor glimpses of 'Sleep Experiments' going around. \
-		One thing is certain though, a rise of gibbers on-board Russian stations, which was followed by a rise in Beefmen. \
-		No one knows what happened during this era, but when the program finally ended and they were allowed into society, \
-		none of them were able to live 'normally'. Their complete inability to sleep, but their power in traversing the 'Phobetor Tear' \
-		immediately started popping everywhere, and has been at the	forefront of galaxy-wide investigations of their anatomy and the experiments.",
+		"On a very quiet day the Russian \"Fiddler Diner\" was serving food to its station's crew. Shortly into the lunch rush they ran out of burger meat. \
```

Comment:
The Diner wasn't on a station

Reply:
The original text indicates that the diner was serving food to "the crew" and the cook was getting a corpse from "the morgue". That seems to imply that the story is taking place on a singular station rather than its setting either having a traveling band of chefs or being a city floating through the void of space.

Reply:
The story indicates that the diner was serving food to "the crew" and the chef was getting a body from "the morgue". This seems to indicate that the setting was a singular station.

Reply:
The story of the first beefman predates space stations which is why its the russian sol government that found it

Reply:
The "Russian Sol Governement" would have to be a government on the level of Sol's planetary system or else it would've just been the regular Russian government. If there's a government on the level of our solar system then the existence of space stations is almost guaranteed.

Reply:
Also the original story literally mentions space stations in this same segment:

> "After a quick confiscation from the Russian Sol Government, most records of 'Beefmen' have gone dark, \ with minor glimpses of 'Sleep Experiments' going around. \ One thing is certain though, a rise of gibbers on-board Russian stations, which was followed by a rise in Beefmen."

Reply:
It's a mistake so please fix that instead

Reply:
Removing every allusion to space stations from the original text would be fairly difficult. It would involve rewriting far more than what this proofreading has already altered. The notion of that being within the scope of this PR is questionable at best, but very well…

…If I am to begin doing this, however, then I must know one thing:

Where does this story take place at? Aside from a space station, the only other setting that has a kitchen, a crew, and a morgue would have to be something to the effect of a _cruise ship_.